model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/InfinispanUserSessionProvider.java

```diff
@@ -270,6 +336,14 @@ private UserSessionEntity getUserSessionEntity(RealmModel realm, String id, bool
         return entity;
     }
 
+    private Stream<UserSessionModel> getUserSessionsFromPersistenceProviderStream(RealmModel realm, UserModel user, boolean offline) {
+        UserSessionPersisterProvider persister = session.getProvider(UserSessionPersisterProvider.class);
+        return persister.loadUserSessionsStream(0, Integer.MAX_VALUE, realm, user, offline)
```

Comment:
I think it is good to have pagination support here, but I wasn't sure about default limits.

Reply:
IMO pagination support may not be so urgent for the users? I suppose that it will be very rare that single user has for example 1000  offline sessions? Pagination makes sense for the clients IMO and it is already available in the admin console (however not yet supported in the PR as mentioned in the comment).